.github/workflows/build_extension.yml

```
@@ -0,0 +1,161 @@
+          pytest -vv -r ap --cov ${{ steps.get_python_name.outputs.python_name }}
+
+          jupyter labextension list
+          jupyter labextension list 2>&1 | grep -ie "${{ inputs.extension }}.*OK"
+          python -m jupyterlab.browser_check
+
+      - name: Package the extension
+        run: |
+          set -eux
+
+          pip install build
+          python -m build          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)